Tests/KituraContractsTests/QueryCoderTests.swift

```diff
@@ -18,14 +18,25 @@ import XCTest
 
 @testable import KituraContracts
 
+@available(OSX 10.12, *)
 class QueryCoderTests: XCTestCase {
 
     static var allTests: [(String, (QueryCoderTests) -> () throws -> Void)] {
         return [
             ("testQueryDecoder", testQueryDecoder),
             ("testQueryEncoder", testQueryEncoder),
+            ("test1970Decode", test1970Decode),
+            ("test1970Encode", test1970Encode),
+            ("testISODecode", testISODecode),
+            ("testISOEncode", testISOEncode),
+            ("testCustomDecode", testCustomDecode),
+            ("testCustomEncode", testCustomEncode),
+            ("testFormattedDecode", testFormattedDecode),
+            ("testFormattedEncode", testFormattedEncode),
+            ("testCustomArrayDecode", testCustomArrayDecode),
+            ("testCustomArrayEncode", testCustomArrayEncode),
             ("testCycle", testCycle),
-            ("testIllegalInt", testIllegalInt)
+            ("testIllegalInt", testIllegalInt),
```

Comment:
If we retain the ability to customize the default `QueryEncoder.dateEncoder` / `QueryDecoder.dateDecoder`, then we should have a test that demonstrates that it works (where the QueryParams does not explicitly set a strategy).